### ISSUE TITLE
Vignette not working with install(build_vignettes = TRUE)

### ISSUE THREAD

**Issue by andrew-edwards** [andrew-edwards/eDNAcutoff#8]
Clean R session, this works:
```
load_all()
build_vignettes()
```
but 
```
install(build_vignettes = TRUE)
```
fails and can't find `small_example`.

**Comment by andrew-edwards** [comment]
Clean R, 
```
build_vignettes()
```
does not work - I forget if it should though.

**Comment by andrew-edwards** [comment]
@seananderson - any obvious thing I'm doing wrong in the above? (Don't worry about looking at the code, just wondered if I'm missing a step). I did screw up something last week (detached HEAD etc.) and it was working before then, but I've remade the data/*.Rda files and they are in there, including `small_example`. Thanks. (I've now set up Travis, though I knew it didn't work before doing that).

**Comment by andrew-edwards** [comment]
Awesome, thanks @seananderson.  I tried library() a few times, and vignette had worked in the past but I think because I had already installed it. And, yes, I forgot to export functions (which hadn't mattered before I think), and maybe hadn't done `build()` enough. Thanks again.

Sean said (in pull request, putting here): You just need to library() your package and export a couple functions. The vignette builds from the installed package, not from the load_all() temporary version.
